functional-test/src/test/groovy/org/rundeck/util/gui/pages/project/DashboardPage.groovy

```diff
@@ -11,14 +11,17 @@ import org.rundeck.util.gui.pages.BasePage
 import java.time.Duration
 
 @CompileStatic
-class DashboardPage extends BasePage{
+class DashboardPage extends BasePage {
 
     String loadPath = ""
     By projectDescriptionBy = By.className("text-project-description")
     By projectLabelBy = By.xpath("//div[@data-ko-bind='projectHome']")
     By readmeMarkDownBy = By.className("markdown-body")
     By projectSummaryBy = By.id("projectHome-summary")
     By projectSummaryCountLinkBy = By.cssSelector("#projectHome-summary a .summary-count")
+    By executionCountBy = By.xpath("/html/body/section[1]/div/section/div[2]/div/div/div[2]/div/div/div/div/div/a")
+    By userCountBy = By.xpath("//p[contains(text(),'by')]/span[@class='text-info']")
+    By userElementBy = By.xpath("/html/body/section[1]/div/section/div[2]/div/div/div[2]/div/div/div/div/div/div/ul/li")
```

Comment:
this locator is way to hard to read and locate. Instead try to use locators by classname, "users" perhaps would provide the same result